anime/views.py

```diff
@@ -44,3 +44,32 @@ def get_anime_by_name(name: str):
         return anime_schema.dump(row)
     else:
         abort(404)
+
+
+@animes_bp.route('/api/v1/animes/<int:id>', methods=['PUT'])
+def anime(id: int):
+    """Route updates attribute(s) of single anime in DB
+
+    :param name: Anime name to update its attribute(s)
+    :type name: str
+    :return: Query paremeters
+    :rtype: JSON
+    """
+
+    try:
+        request_data = request.get_json()
+
+        for key, val in request_data.items():
+            db.session.query(Anime).filter(Anime.anime_id == id).update(
+                {key: val}
+            )
+            db.session.commit()
+
+        anime = db.session.execute(
+                    db.select(animes_tab).filter_by(anime_id=id)).first()
```

Comment:
I don't feel like this part is needed. We should be able to get the updated anime object from db.session......update(). Could we try that?